### PR TITLE
Feature:Social Login 

### PR DESCRIPTION
**Pull request by polatkanpolat5**
i made it

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (polatkanpolat5) [inline file3.txt]: This line is good, but it's better you change it to hellom

**Review by polatkanpolat5 — COMMENTED** [review]
please make it hellom

File: file3.txt (hunk 1 above)
Comment (polatkanpolatdev) [inline file3.txt]: yes make it hellom

**Review by polatkanpolatdev — CHANGES_REQUESTED** [review]
helloom

**Review by polatkanpolatdev — APPROVED** [review]

**Pull request merged**